docs/src/developing/guides/compressed-nfts.md

```diff
@@ -282,7 +282,7 @@ two related instructions:
 2. actually create the tree, owned by the Bubblegum program
 
 Using the
-[`createAllocTreeIx`](https://solana-labs.github.io/solana-program-library/account-compression/sdk/docs/modules/index.html#createAllocTreeIx)
+[`createAllocTreeIx`](https://github.com/solana-labs/solana-program-library/blob/e4908b929f69f108f978b586bd0cd5b43f24cb1e/account-compression/sdk/src/instructions/index.ts#L169)
```

Comment:
This is actually meant to the link to the typedocs, which requires removing the `docs/` part of the path, ie

```suggestion
[`createAllocTreeIx`](https://solana-labs.github.io/solana-program-library/account-compression/sdk/modules/index.html#createAllocTreeIx)
```